platform/util/platform-util/src/test/java/org/codice/ddf/platform/util/InputValidationTest.java

```diff
@@ -95,4 +114,30 @@ public void testCheckForClientSideVulnerableMimeTypeGood() {
         boolean result = InputValidation.checkForClientSideVulnerableMimeType(GOOD_MIME);
         assertTrue(result);
     }
+
+    @Test
+    public void testSanitizeFilenameKnownBadCaseInsensitive() {
+        String sanitizedName = InputValidation.sanitizeFilename(BAD_FILE3);
+        assertThat(sanitizedName, is(DEFAULT_FILE));
```

Comment:
Could also use `assertThat("This is my failure reason", sanitizedName, is(DEFAULT_FILE));` if you still want to use matchers